browsers/deno.json

```diff
@@ -0,0 +1,100 @@
+{
+  "browsers": {
+    "deno": {
+      "name": "Deno",
+      "releases": {
+        "1.0": {
+          "release_date": "2020-05-13",
+          "release_notes": "https://github.com/denoland/deno/releases/tag/v1.0.0",
+          "status": "retired",
+          "engine": "V8",
+          "engine_version": "8.4"
```

Comment:
Where could I find an official and parsable source of information about matching V8 versions to Deno versions, like https://nodejs.org/dist/index.json ?

Reply:
We do not have that information in programmatic form right now, but I have opened https://github.com/denoland/deno/issues/11195 to track creation of such a file. If you need info about a specific file, V8 versions are printed when running `deno --version`, and are available at `Deno.version.v8` at runtime. You can also upgrade and downgrade to any version using `deno upgrade --version=$version`, and you can find a list of all versions at https://raw.githubusercontent.com/denoland/deno_website2/main/versions.json.